cartography/intel/aws/securityhub.py

```diff
@@ -25,7 +25,7 @@ def get_hub(boto3_session: boto3.session.Session) -> Dict:
 
 def transform_hub(hub_data: Dict) -> None:
     if 'SubscribedAt' in hub_data and hub_data['SubscribedAt']:
-        subbed_at = datetime.datetime.strptime(hub_data['SubscribedAt'], "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
Under the old method, the test expected `1606993517` but my laptop returned `1607022317`.